src/Player.cpp

```diff
@@ -58,4 +58,17 @@ void Player::setGlobalBottom(float bottom)
     setPosition(bounds.left, y);
 }
 
-} // namespace tt
\ No newline at end of file
+
+void Player::addItem(const std::string& itemID)
+{
+    _inventory[itemID]++;
+}
+
+
+std::map<std::string, std::int32_t> Player::getInventory()
```

Comment:
I generally wouldn't return a map like this since this will copy the entire map (and everything in it). Instead you can return a read/write reference:
```
std::map<...>& Player::getInventory()
```

Or a read-only reference
```
const std::map<..>& Player::getInventory() const
```
I would err on the side of returning the read-only version. Notice I added `const` at the beginning and the end since this is also a `get`ter function.

I have a follow-up comment to this down where you call this function.